make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import github
+import requests
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+WORKING_DIR = "./tmp"
+DIFF_FILE = "./diff.patch"
```

Comment:
It seems like this could also be a named temporary file, which would be cleaner.

Reply:
Similar to the previous comment on the working dir, I prefer to keep it in the same working directory so that it shows up in the IDE for debugging purposes.